docs/docs/locales.md

```diff
@@ -94,13 +94,13 @@ To learn more about the Config command, visit https://docs.transifex.com/client/
 
 ### Pulling translation updates from Transifex
 
-Pull acceptable translation files with 16 threads:
+Pull acceptable translation files with 16 threads using `git` timestamps (else it'll use filesystem timestamps which seems completely wrong):
 
 ```
-tx pull --minimum-perc 95 --workers 16
+tx pull --minimum-perc 70 --workers 16 --use-git-timestamps
```

Comment:
95% is too strict IMO

Reply:
actually it doesn't matter really, it'll just pull & update whatever's locally available. `--minimum-perc` is probably only important when pulling _new_ files in batches.